<launch>
	<!-- Start navigation -->
    <include file="$(find pioneer_cyton_2dnav)/move_base.launch" />

	<!-- start Moveit -->
	<include file="$(find pioneer_cyton_moveit)/launch/move_group.launch">
		<arg name="info" value="true"/>
	</include>
	
	<!-- Start Perception -->
	<node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" output="screen">
		<rosparam command="load" file="$(find pioneer_cyton_gazebo)/config/simple_grasping.yaml" />
	</node>

	<!-- Drive to the table, pick stuff up -->
	<node name="demo" pkg="pioneer_cyton_gazebo" type="demo.py" output="screen"/> 

</launch>
